include/simdjson/portability.h

```diff
@@ -15,6 +15,43 @@
 #define IS_ARM64 1
 #endif
 
+#if defined(__clang__) && defined(_MSC_VER) && defined( IS_X86_64)
```

Comment:
Oh Jesus this if makes my brain hurt :)

Reply:
You can actually try this out easily. Get Visual Studio, install clang for it. And then you get this hybrid. It actually works surprisingly well. But for us, it is a bit tricky.